src/hooks/product/use-product-handlers.js

```diff
@@ -1,12 +1,12 @@
 import { useState, useMemo, useCallback } from 'react';
 import { useSelector } from 'react-redux';
 import { config } from '../../configs';
-import {
-  AdditionsSelector,
-  OptionsSelector
-} from '../../redux/selectors/use-product-handlers.selectors';
 import useProductSpecies from './use-product-species';
 
+const AdditionsSelector = ({ additions }) => additions.length;
+
+const OptionsSelector = ({ options }) => options.find(AdditionsSelector);
```

Comment:
It's make sense to rename it to `selectOptionsLength`

Reply:
fixed